adapters/ix/ix_test.go

```diff
@@ -253,6 +253,9 @@ func TestBuildIxDiag(t *testing.T) {
 
 	for _, test := range testCases {
 		t.Run(test.description, func(t *testing.T) {
+			// save and later restore the value of the global variable to avoid side-effecting other tests
+			oldVersionVer := version.Ver
+			defer func() { version.Ver = oldVersionVer }()
 			version.Ver = test.pbsVersion
```

Comment:
I'm not sure how this made it through PR review, but we definitely do not want to alter global non-adapter state during tests. Instead, please extract the version formatting to a separate function and have the unit tests target that function specifically.